src/STM32RTC.cpp

```diff
@@ -140,12 +140,20 @@ void STM32RTC::setClockSource(Source_Clock source)
   * @param  predivS: pointer to the current Synchronous prescaler value
   * @retval None
   */
+#if defined(STM32F1xx)
+void STM32RTC::getPrediv(int32_t *predivA, int16_t *predivS)
+{
+  UNUSED(predivS);
+  RTC_getPrediv(predivA, 0);
```

Comment:
```suggestion
void STM32RTC::getPrediv(uint32_t *predivA, int16_t *dummy)
{
  UNUSED(dummy);
  RTC_getPrediv(predivA, 0);
```

Reply:
yes, predivS is a dummy param for stm32F1 but then changing the param name does not match the description of the function anymore. So should we keep predivS even if dummy (it is still confirmed by the UNUSED)

Reply:
Well I think it could be documented in the comment as it is specific to STM32F1 explaining it is for comaptibility and update the README.md as well. Using dummy will be more explicit at build time if an error occur.

Reply:
yes, function help modified in that way